src/deckardcain.js

```diff
@@ -1,5 +1,6 @@
 export const API_BLUEPRINT_HEADER = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))\s*?([\n\r]{1,2}|$))/i;
-export const API_BLUEPRINT_RESPONSE = /\+\s+(?:response|request)\s+\d{3}/i;
+export const API_BLUEPRINT_RESPONSE = /[-\*\+]\s+(response|request|attributes?)/i;
```

Comment:
Here we're adding a few new variants to the list items `- response` and `* response` etc. I didn't see any tests for them can you please add additional test cases.